metainfo.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.Vysp3r.ProtonPlus</id>
```

Comment:
this is case sensitive and should match the app-id declared in the manifest.

Reply:
fixed it